core/commands/pin/remotepin.go

```diff
@@ -173,6 +175,34 @@ NOTE: a comma-separated notation is supported in CLI for convenience:
 		if err != nil {
 			return err
 		}
+		//ifconfig.co
+		checkConnection := func() bool {
+			_, err := http.Get("ifconfig.co")
+			return err == nil
+		}
```

Comment:
`checkConnection` needs to be removed.

We can't use a third party DNS name + HTTP service for doing this test.
If it goes offline or can't be reached, our _online_ users will get a false-negative.

Checking `node.IsOnline` is the best we can do without introducing centralized services.